### DISCUSSION TITLE
Tags next to maps

### DISCUSSION THREAD

**Discussion by GiveMeFox** [ppy/osu#22811] [Ideas]
Tags appearing before the name of the map
tags would either be a separate feature or use collections so a map would display the collection it's in as its tag
it would also be nice to make it toggleable for people that don't want it
if a map is in more than one collection the tags will overlap each other only showing one of them
picking colors for them using RGB hex or any other method would be nice to have if they got implemented
it could also be possible to pick a fixed tag for a map

**Comment by peppy** [comment]
Most beatmaps have tens or tags. Are you proposing a custom tag system similar to collections where you, the user, is defining the tags?

**Reply by GiveMeFox** [reply]
yes
the color of tags could maybe use the color scheme of a skin thats currently being used or tags could be a part of skins :o

**Reply by GiveMeFox** [reply]
![osu_tags](https://user-images.githubusercontent.com/74460065/223778848-0f8e7184-0bd1-4ccc-9c2f-ab337da7f8ba.svg)
perhaps something like this but with text inside :o or no text at all
